### PR TITLE
feat(edl): highlights for EDL

### PR DESCRIPTION
**Pull request by mrdgo**
EDL is intel's enclave definition language to define SGX enclaves. These files probably never need anything more than highlights.

### CODE REVIEW

**Comment by clason** [comment]
Well, they do need a parser, otherwise they are worthless...

**Comment by mrdgo** [comment]
> Well, they do need a parser, otherwise they are worthless...

[not mine](https://github.com/Andrade/tree-sitter-edl)

**Comment by clason** [comment]
Well, so, you need to add this parser here. Otherwise the queries won't work.

File: lua/nvim-treesitter/parsers.lua [hunk 1]
```diff
@@ -466,6 +466,15 @@ list.editorconfig = {
   maintainers = { "@ValdezFOmar" },
 }
 
+list.edl {
+  install_info = {
+    url = "https://github.com/Andrade/tree-sitter-edl",
+    files = { "src/parser.c" },
+    branch = "master",
```
Comment (clason) [inline lua/nvim-treesitter/parsers.lua:473]: Not needed unless it's a non default branch

**Review by clason — CHANGES_REQUESTED** [review]

**Comment by mrdgo** [comment]
happy to remove the branch. But I am not sure if I want to fight that pipeline

**Comment by clason** [comment]
> happy to remove the branch. But I am not sure if I want to fight that pipeline 

It's not optional, though. Quality control is important. If you are not willing to make the effort (and keep doing so when updates are required), it's better to close the PR now to avoid wasting both our time.

**Comment by mrdgo** [comment]
> Also, never open a PR from your master branch. Create a feature branch first, make your changes, and then open a PR.

history beautification reasons, right?

**Comment by clason** [comment]
No, so CI works properly and maintainers can push to the PR to fix issues such as the current ones.

**Pull request closed without merging**